SimTracker/TrackerMaterialAnalysis/test/listIds_PhaseII.py

```diff
@@ -4,16 +4,7 @@
 
 process = cms.Process("MaterialAnalyser")
 
-readGeometryFromDB = False
-
-if not readGeometryFromDB:
-  process.load('Configuration.Geometry.GeometryExtended2026D41Reco_cff')
-else:
-# GlobalTag and geometry via GT
-  process.load('Configuration.Geometry.GeometrySimDB_cff')
-  process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
-  from Configuration.AlCa.GlobalTag_condDBv2 import GlobalTag
-  process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic', '')
```

Comment:
on second thought, the phase-2 reco geometry will eventually come from DB as well. Can this change be reverted?

Reply:
Hi Marco, I think we can wait for that option to be available and then add the necessary functionality as needed. The *eventually* part has been there since 2016 :sweat_smile: See:  https://github.com/cms-sw/cmssw/blob/c53ec47424bd5e5c7fb2eafb31c06cb579d0287d/SimTracker/TrackerMaterialAnalysis/test/trackingMaterialAnalyser_ForPhaseII.py#L9

Reply:
I think we are actually approaching a period in which this should happen, so I would rather leave it :)

Reply:
Sure that can be done the same way we did with `nEvents`, will address this later today :+1: